crates/ruff/src/rules/flake8_annotations/rules/definition.rs

```diff
@@ -432,20 +434,46 @@ fn is_none_returning(body: &[Stmt]) -> bool {
 
 /// ANN401
 fn check_dynamically_typed<F>(
+    checker: &Checker,
     annotation: &Expr,
     func: F,
     diagnostics: &mut Vec<Diagnostic>,
-    is_overridden: bool,
```

Comment:
I've moved the `is_overriden` check outside the function as it seems common. This avoids having many arguments to a function.